packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -76,14 +76,28 @@ const Selector = styled.div`
 
             ${
               error && !isOpen
-                ? `border-color: ${colors.feedback.attention[1]};`
+                ? `border-color: ${colors.feedback.attention[1]};
+                label {
+                  color:${colors.feedback.attention[1]}; 
+                };
+                `
                 : ''
             }
           }
-          &:hover {
-            fieldset {
-              border-color: ${dropdown.hover.selector.border.color};
-            }
+          ${
+            error && !isOpen
+              ? `
+              &:hover {
+              fieldset {
+                border-color: ${dropdown.hover.selector.border.error};
+              }
+            }`
+              : ` 
+              &:hover {
+              fieldset {
+                border-color: ${dropdown.hover.selector.border.color};
+              }
+            }`
           }
```

Comment:
Maybe we could move the conditional to where the change happens. Would this work?

```suggestion
          &:hover {
            fieldset {
              border-color: ${
                error && !isOpen
                  ? dropdown.hover.selector.border.error
                  : dropdown.hover.selector.border.color
              };
            }
          }
```

Reply:
It works, and seem so much clear!